code/modules/research/designs/machine_designs.dm

```diff
@@ -460,4 +460,10 @@
 	build_type = IMPRINTER
 	materials = list(MAT_GLASS = 1000, "sacid" = 20)
 	build_path = /obj/item/weapon/circuitboard/grounding_rod
-	category = list ("Misc. Machinery")
\ No newline at end of file
+	category = list ("Misc. Machinery")
+
+/datum/design/bot_upgrade_boost
+	name = "Bot Upgrade: Boost"
+	desc = "The circuit board for giving a speed boost to bots"
+	id = "bot_upgrade_boost"
+	req_tech = list("programming" = 2, "engineering" = 2)
```

Comment:
I've moved all of this information into a new file as we talked about, and now it appears under the "Misc" section along with things like batteries